src/components/NewEntry.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import NewEntry from './NewEntry'
+
+export default {
+  title: 'Component/NewEntry',
+  component: NewEntry,
+}
+
+const Template = (args) => <NewEntry {...args} />
+
+export const form = Template.bind({})
```

Comment:
I think you write `Form` here and not `form`. 🤔

Reply:
Because of the styled component? 🤔